envoy/upstream/upstream.h

```diff
@@ -1060,9 +1061,12 @@ class ClusterInfo : public Http::FilterChainFactory {
   virtual ClusterEndpointStats& endpointStats() const PURE;
 
   /**
-   * @return ClusterTrafficStats& all traffic related stats for this cluster.
+   * @return LazyClusterTrafficStats&  all traffic related stats for this cluster.
```

Comment:
not necessary to give the return type in the Doxygen comment -- it's in the code right below.

Reply:
That's the pattern in this whole file.